### PR TITLE
zustand added

### LINKED ISSUE #24: state management

### PR DESCRIPTION
**Pull request by Somidh**
Closes #24 

Please review it and check if all okay or not

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
setTweets, setDates, setUsername, setUserProfile as well

**Comment by Somidh** [comment]
I have added those changes, can u please check and review it

File: app/tweetStore.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+const useTweetStore = create<StoreValues>((set, get) => ({
+    // by using get u can get the values from state or the whole state but the set function receives a second parameter as a function which is use to get the previous state 
+    //  set(state => ({...state, username: newUsername}))
+    dates: [],
+    tweets: [],
+    username: "",
+    userProfile: { name: "", image: "" },
+
+    // This is an example, u can remove this below function 
+    userName: (name: string) => {
```
Comment (Aadarsh805) [inline app/tweetStore.ts:26]: why's there two userName set functions

**Comment by Aadarsh805** [comment]
let me merge and try this

**Pull request merged**